modules/cli/cmds/apply_test.go

```diff
@@ -0,0 +1,97 @@
+package cmds
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/rancher/fleet/modules/cli/apply"
+	"github.com/rancher/fleet/pkg/bundlereader"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	username = "user"
+
+	password_file    = "password_file"
+	password_content = "pass"
+
+	caCerts_file    = "caCerts_file"
+	caCerts_content = "caCerts"
+
+	sshPrivateKey_file    = "sshPrivateKey_file"
+	sshPrivateKey_content = "sshPrivateKey"
+
+	helmSecretsNameByPath_file = "helmSecretsNameByPath_file"
+)
+
+var helmSecretsNameByPath_content = map[string]bundlereader.Auth{"path": {Username: username, Password: password_content}}
+
+func TestAddAuthToOpts(t *testing.T) {
+	tests := map[string]struct {
+		name         string
+		apply        Apply
+		expectedOpts *apply.Options
+		expectedErr  error
+	}{
+		"Auth is empty if no arguments are provided": {
+			apply:        Apply{},
+			expectedOpts: &apply.Options{},
+			expectedErr:  nil,
+		},
+		"Auth contains values from username, password, caCerts and sshPrivatey when helmSecretsNameByPath not provided": {
+			apply:        Apply{PasswordFile: password_file, Username: username, CACertsFile: caCerts_file, SSHPrivateKeyFile: sshPrivateKey_file},
+			expectedOpts: &apply.Options{Auth: bundlereader.Auth{Username: username, Password: password_content, CABundle: []byte(caCerts_content), SSHPrivateKey: []byte(sshPrivateKey_content)}},
```

Comment:
nit: I think that adding line breaks here (eg. one struct member per line) could make these test case specs easier to read. How do you see it?

Reply:
I think it is easier to read it this way. I'm happy to change it if you all feel it is better in multiple lines.

Reply:
No worries then, let's keep it this way.